ai-ml/model-train/terraform/gke-standard/main.tf

```diff
@@ -0,0 +1,74 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# create private subnet
+module "network" {
+  source         = "../modules/network"
+  project_id     = var.project_id
+  region         = var.region
+  cluster_prefix = var.cluster_prefix
+}
+
+# [START gke_model_train_standard_private_regional_cluster]
+module "training_cluster" {
```

Comment:
**Question:** Will the Terraform (from the repo) be embedded in the tutorial as well?

Reply:
What do you mean by "embedded"? In the guide we run "git clone" and "terraform apply" with this code